rpc/auth.go

```diff
@@ -256,18 +270,6 @@ type Credentials struct {
 	Payload string          `json:"payload"`
 }
 
-// RSAPublicKeyThumbprint returns SHA1 of the public key's modulus Base64 URL encoded without padding.
-func RSAPublicKeyThumbprint(key *rsa.PublicKey) (string, error) {
```

Comment:
we should deprecate instead of removing - app is using this method.

Reply:
it's using it via goutils/rpc, not app

Reply:
ah right